dm/syncer/syncer.go

```diff
@@ -691,6 +693,9 @@ func (s *Syncer) Process(ctx context.Context, pr chan pb.ProcessResult) {
 		<-newCtx.Done() // ctx or newCtx
 	}()
 
+	s.running.Store(true)
```

Comment:
i think this part should put into `syncer.Run`

Reply:
the name `syncer.Run` is actually misleading, `Process` is the real `run` which is called in a a separate routine, `syncer.Run` is something like `doRun` which is only called inside `process`. so i put it here.